api/src/main/java/org/pmiops/workbench/workspaces/resources/UserRecentResourceServiceImpl.java

```diff
@@ -120,19 +126,37 @@ public void deleteNotebookEntry(long workspaceId, long userId, String notebookPa
         notebookPath);
   }
 
-  /** Deletes cohort entry from user_recently_modified_resource */
+  /**
+   * When a Cohort recent-resource is deleted, delete all Cohort Reviews and Datasets which
```

Comment:
please add the text "entry" here to make it clear we're not removing the actual resources